pkg/sql/compile/operator.go

```diff
@@ -487,6 +487,7 @@ func constructInsert(n *plan.Node, eg engine.Engine, proc *process.Process) (*in
 		UniqueIndexDef:       uDef,
 		SecondaryIndexTables: secondaryIndexTables,
 		SecondaryIndexDef:    sDef,
+		ClusterByDef:         n.TableDef.ClusterBy,
```

Comment:
This modification needs to modify the proto file at the same time, and insert is going to be transferred remotely, so you need to consider this situation.